gst/tensor_decoder/tensordec.c

```diff
@@ -543,7 +558,7 @@ gst_tensordec_get_property (GObject * object, guint prop_id,
       g_value_set_boolean (value, self->silent);
       break;
     case PROP_MODE:
-      g_value_set_string (value, self->mode);
+      g_value_set_uint (value, self->mode);
```

Comment:
It's supposed to be string.

Having uint/int value for ```self->mode``` is great (it's highly recommended to do so), however, the get_property needs to return the same type.